modules/seqexec/server/src/main/scala/seqexec/server/StepType.scala

```diff
@@ -13,7 +13,6 @@ sealed trait StepType {
 
 object StepType {
   final case class CelestialObject(override val instrument: Instrument) extends StepType
-  final case class Dark(override val instrument: Instrument) extends StepType
```

Comment:
No darks?

Reply:
The type was not used anywhere. `DarkOrBias` was used instead.